docs/concepts/event-driven-architecture.md

```diff
@@ -35,17 +35,18 @@ To implement event-driven architecture in your Amplication project, you'll need
 
 1. Create a message broker
 2. Define topics
-3. Connect services to the message broker
-4. Install the Kafka plugin
-5. Implement event production and consumption logic
+3. Connect services to the message broker and relevant topics
+4. Implement event production and consumption logic
 
 :::info
 For a step-by-step guide on building an event-driven application with Amplication and Kafka, refer to the blog post: [Building Real-time Applications with Amplication and Kafka](https://amplication.com/blog/building-real-time-applications-with-amplication-and-kafka).
 :::
 
 ## Step 1: Create a Message Broker
 
-In your Amplication project, navigate to "Add Resource" and select "[Message Broker](/how-to/create-message-broker)". This will lead you to the Message Broker Creation Wizard. Configure the settings for your message broker, such as the broker type (e.g., Kafka) and connection details.
+In your Amplication project, navigate to "Add Resource" and select "[Message Broker](/how-to/create-message-broker)". This will lead you to the Message Broker Creation Wizard. 
+
+After creating the message broker, you'll be able to configure the settings by installing and configuring the appropriate [message broker plugin](#supported-message-brokers).
```

Comment:
That's good! However, as mentioned before- I would give it more emphasize as part of the flow, as it may be confusing for some people (- the need to add resource + install relevant plugin